infra/spawn_data_pipeline.py

```diff
@@ -105,6 +107,14 @@ def main():
 
         payload["dependencies"] = new_dependencies
 
+        # Override the Docker image tag if needed
+        if docker_tag:
+            base_image = payload["payload"]["image"]
+            tagless_image = base_image.rsplit(":", 1)[0]
```

Comment:
Let's actually assert the tag doesn't exist.

Reply:
Done, I have allowed the tag `latest` as it's more or less the equivalent of no tag

Reply:
:+1: